composer.json

```diff
@@ -67,6 +67,9 @@
         ]
     },
     "config": {
+        "platform": {
```

Comment:
I think we should stop committing the lock file (Core SE is about to remove it again as well).